test/functionalTest/cases/947_v2_entities/v2_entities_simple.test

```diff
@@ -0,0 +1,426 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Probably I haven't expressed correctly... I was referring all the time to the directory name. It is ok to keep the .test file name as it is now (GET_v2_entities.test).

I men, what I'm proposing to change is:

cases/947_v2_entities/

to 

cases/947_v2_list_all_entities/